FSharp.Profiles.props

```
@@ -14,4 +14,8 @@
     </Otherwise>
   </Choose>
 
+  <PropertyGroup>
+     <!-- Override the setting for the Arcade UserRuntimeConfig for fsc on .NET Core -->
+    <UserRuntimeConfig Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(OutputType)' == 'Exe' AND '$(AssemblyName)' == 'fsc'">$(MSBuildThisFileDirectory)src\fsc\runtimeconfig.template.json</UserRuntimeConfig>
```

Rather than creating your own runtimeconfig file, you can just use the `ServerGarbageCollection` property: https://learn.microsoft.com/en-us/dotnet/core/runtime-config/garbage-collector#workstation-vs-server

```suggestion
    <ServerGarbageCollection>true</ServerGarbageCollection>
```